### PR TITLE
Add documentation for multi-domains certificates

### PR DESCRIPTION
**Pull request by tgalopin**
RFR @jderusse

### CODE REVIEW

File: cli/get-multidomain-certificate.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+currentMenu: cli-get-multidomain-certificate
+---
+
+# Multi-domains certificates
+
+Let's Encrypt and most ACME servers are able to provide multi-domain certificates. Such certificates
+will be usable for multiple domains as a single file, which can be useful in many cases (for instance to
+use the same certificate for `yourdomain.org` and `www.yourdomain.org`).
+
+**Note:** Multi-domains certificates use a technology called [Server Name Indication](https://en.wikipedia.org/wiki/Server_Name_Indication)
```
Comment (jderusse) [inline cli/get-multidomain-certificate.md]: Not exactly: In SNI, the client send the hostname ans the server _select_ the best certificate who fit with the domain.

The multi-domains certificate generate (as you describe very well) 1 certificate with several subjects. The server does not have to _choose_ a certificate, it just send the big certificate which contains every thing.

SANs is pretty old and well supported by browsers (https://www.digicert.com/subject-alternative-name-compatibility.htm)

File: cli/get-multidomain-certificate.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+The only difference is when you use the `request` command, to add the other domains you want to support:
+
+``` console
+php acmephp.phar request yourdomain.org -a www.yourdomain.org -a anotherdomain.org
+```
+
+**Note**: you need to be the authorized owner of all the domains you request this way. In this case, it means
+you have to run `php acmephp.phar authorize yourdomain.org`, `php acmephp.phar authorize www.yourdomain.org`
+and `php acmephp.phar authorize anotherdomain.org` before being able to request the multi-domains certificate.
```
Comment (jderusse) [inline cli/get-multidomain-certificate.md]: don't forget the `check` command too

File: cli/get-multidomain-certificate.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+The process will be the same as with a simple certificate: the command will ask you some informations, request
+the certificate to the ACME server and store it under the `~/.acmephp` directory.
+
+The first domain provided will be the main one for Acme PHP, meaning the multi-domain certificate will be stored
```
Comment (jderusse) [inline cli/get-multidomain-certificate.md]: Technicaly not the first, given there is 1 Domain and N alternativeDomains.

By the ways, that's means, we can't manage 2 certificates for the same domain with differents alternative names. The last generated will override the previous

File: cli/get-multidomain-certificate.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+under this domain. Therefore, in our example, 6 files will be created in the storage directory:
+  
+- **The full-chain certificate** at `~/.acmephp/master/certs/yourdomain.org/fullchain.pem`
+- **The certificate private key** at `~/.acmephp/master/private/yourdomain.org/private.pem`
+- **The chain alone** at `~/.acmephp/master/certs/yourdomain.org/chain.pem`
+- **The certificate alone** at `~/.acmephp/master/certs/yourdomain.org/cert.pem`
+- **The combined certificate** at `~/.acmephp/master/certs/yourdomain.org/combined.pem`
+- **The certificate public key** at `~/.acmephp/master/private/yourdomain.org/public.pem`
```
Comment (jderusse) [inline cli/get-multidomain-certificate.md]: May we group _private_ files and _certs_ files together

File: cli/get-multidomain-certificate.md (hunk 1 above)
Comment (tgalopin) [reply cli/get-multidomain-certificate.md]: Thanks, I removed the note

**Comment by tgalopin** [comment]
Comments adressed.

**Review by jderusse — APPROVED** [review]
I would talk about SANs to explain how multi domain works, but it's also ok like this.

**Pull request merged**